src/chains/ethereum/ethereum/src/data-managers/block-manager.ts

```diff
@@ -272,26 +272,45 @@ export default class BlockManager extends Manager<Block> {
     await this.#blockIndexes.put(LATEST_INDEX_KEY, number);
   }
 
+  async getEarliest() {
+    const fallback = this.#blockchain.fallback;
+    if (fallback) {
+      const json = await fallback.request<any>(
+        "eth_getBlockByNumber",
+        [Tag.earliest, true],
+        { disableCache: true }
```

Comment:
```suggestion
        // TODO: re-enable cache once this is fixed
        // https://github.com/trufflesuite/ganache/issues/3773
        { disableCache: true }
```
I see the reasoning for the disabled cache now, and think it's worth moving forward on this fix without also fixing the cache.